bios/students/emily.md

```diff
@@ -0,0 +1,7 @@
+![alt text] (https://avatars2.githubusercontent.com/u/25447394?v=4&s=460)
+# Emily Hannan
+
+* __Favourite food__: blackened salmon over sticky white rice. 
+* __Experience with technology: live/breathe photoshop, love/hate relationship with google docs + dropbox, and can edit some wordpress websites. 

```

Comment:
You're missing hte closing ___ (double underscore) after experience technology here, which is going to make your styling go weird... I'll merge it in for now but you might want to change it!

Reply:
Awesome! Thanks  - didn't notice that error.